lib/kc_elixir_web/controllers/page_controller.ex

```diff
@@ -1,7 +1,49 @@
 defmodule KcElixirWeb.PageController do
   use KcElixirWeb, :controller
 
+  @meetups [
+    %{
+      datetime: [
+        ~N{2019-03-07 18:00:00},
+        ~N{2019-03-07 20:00:00}
+      ],
+      image_url: "http://placekitten.com/200/200",
+      topic: "Dialyzer!",
+      location: %{
+        url: "http://www.403club.com",
```

Comment:
Not sure the url is needed for the march meeting. Though the address will need to be switched to:

8500 Shawnee Mission Pkwy
Mission, KS  66202

and venue should be:

Company Kitchen